frontend/src/metabase/dashboard/components/ClickBehaviorSidebar.styled.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled";
+import { darken } from "metabase/lib/colors";
+
+export const SidebarItem = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+`;
+
+export const CloseIconContainer = styled.span`
+  margin-left: auto;
+  padding: 1rem;
+  border-left: 1px solid ${darken("brand", 0.2)};
+`;
+
+export const RemoveIconContainer = styled.span`
+  margin-left: auto;
+  padding: 1.0625rem;
```

Comment:
I know this is just a refactor, but I'm not sure if the extra 1 pixel (`17px` vs `16px`) is justified here. 
![image](https://user-images.githubusercontent.com/1937582/182370451-11b11273-fd5c-4673-91b6-a6fc82614f81.png)

If we can live with the same padding, you could also reuse the container component, as visually, both icons are the same **X** icon. Otherwise, I would think `CloseIconContainer` and `RemoveIconContainer` house different icons.

Reply:
Agree, fixed